### DISCUSSION TITLE
React version of demo page, also looking for recommendation on markdown editor and preview

### DISCUSSION THREAD

**Discussion by ZehuaZhang** [markedjs/marked#2784] [Q&A]
I really like the [demo page](https://marked.js.org/demo/), it's very responsive and the scroll are in sync, is there a react version implementation of that?

Also do you have some recommendations of github or npm packages for feature rich markdown editor with preview using `marked`

**Comment by UziTech** [comment]
I don't know if any markdown preview react components but here is an article on how to create one https://www.freecodecamp.org/news/how-to-build-a-markdown-previewer-with-react-js/
